### DISCUSSION TITLE
[Question] How to make menu separators visible?

### DISCUSSION THREAD

**Discussion by new-sashok724** [tsujan/Kvantum#654] [Q&A] [answered]
Hello again! I would like to make menu separators visible in my theme, but i've not found in documentation or in existing themes how to do it. I've only found option `menu_separator_height` that controls height.

**Comment by tsujan** [comment, marked as answer]
Hi,

`KvBeige` has menu separators (if you don't have an old version of it) and they're explained in `Theme-Making.pdf` → `Indicators` (the table) → `MenuItem`

**Reply by new-sashok724** [reply]
Okay, thanks!
